idaes_examples/notebooks/docs/unit_models/reactors/plug_flow_reactor_src.ipynb

```diff
@@ -557,7 +560,9 @@
    "outputs": [],
    "source": [
     "# Check the degrees of freedom\n",
-    "assert degrees_of_freedom(m) == 0"
+    "assert degrees_of_freedom(m) == 0\n",
+    "# Check units are consistent throughout the model\n",
+    "assert_units_consistent(m)"
```

Comment:
I think we should remove this for now as this is actually not correct (even though it passes); we are not assigning units to the costing coefficients so the units of the costing expressions are not consistent (which would then require us to define a unit for dollars which is a bit of a pain).